samples/arcgis-widget/src/components/Daylight/EsriMapWidget.tsx

```diff
@@ -0,0 +1,102 @@
+import { version } from "esri/kernel";
+import MapView from "esri/views/MapView";
+import SceneView from "esri/views/SceneView";
+import { ComponentModelBase } from "@vertigis/web/models";
+import React, { useEffect, useRef } from "react";
+import { useWatchAndRerender } from "@vertigis/web/ui";
+
+async function injectCssIfNeeded(): Promise<void> {
+    const styleLinkHref = `https://js.arcgis.com/${version}/esri/themes/light/main.css`;
+
+    // Don't inject again if stylesheet already in the document.
+    if (document.querySelector(`link[href="${styleLinkHref}"`)) {
+        return Promise.resolve();
```

Comment:
I think `return;` is more straightforward for void async functions. If you want to match the style of `return new Promise(...)` below, then maybe just omit `async` instead?

Reply:
Good call. I removed the `async` keyword which i think makes it more consistent with the returned promise below.